### PR TITLE
Intro content for Guides and Build It Yourself

### PR DESCRIPTION
**Pull request by ebeneliason**

### CODE REVIEW

**Comment by ebeneliason** [comment]
* [Guides Intro preview link](https://deploy-preview-105--pensive-meitner-faaeee.netlify.app/docs/guides/welcome)
* [Build It Yourself Intro preview link](https://deploy-preview-105--pensive-meitner-faaeee.netlify.app/docs/guides/build-it-yourself/overview/)

File: _docs-sources/guides/welcome.md [hunk 1]
```diff
@@ -3,6 +3,24 @@
...
 
-# Welcome
+import Card from "/src/components/Card"
+import Grid from "/src/components/Grid"
 
-This is an introduction to the guides section
\ No newline at end of file
+# Gruntwork Guides
+
+As a Gruntwork subscriber, you have two primary ways to engage with our library. Whichever approach you choose, our guides provide step-by-step instructions to help you deploy your infrastructure with confidence. We also offer detailed upgrade guides to ensure that your infrastructure remains up-to-date with the latest security patches, compliance requirements, and DevOps best practices.
```
Comment (eak12913) [inline _docs-sources/guides/welcome.md:11]: I think we should be explicit here about what those two ways are: 
- ref arch
- built it yourself

Comment (eak12913) [reply _docs-sources/guides/welcome.md:11]: I guess the buttons are there - but my reptilian brain is still getting hung up on the fact that I didn't get info about what the two ways are right away.

File: _docs-sources/guides/build-it-yourself/1-overview.md [hunk 2]
```diff
@@ -1,5 +1,61 @@
...
 sidebar_label: Overview
+pagination_label: Build It Yourself
 ---
 
+import Card from "/src/components/Card"
+import Grid from "/src/components/Grid"
+
 # Build Your Own Architecture
+
+Now that your foundational knowledge is in place and your workspace is configured, you’re ready to dive in and learn how to deploy production-grade infrastructure. If you’ve purchased a Gruntwork Reference Architecture, use [this guide](/docs/guides/reference-architecture/overview/overview) to get started. Otherwise, view our [courses](/docs/courses) and [guides](/docs/guides/welcome), or check out the [Service Catalog API reference](/docs/reference/services/intro) to learn what's available.
```
Comment (eak12913) [inline _docs-sources/guides/build-it-yourself/1-overview.md:11]: Consider linking `foundational knowledge` and `workspace is configured` to the relevant intro sections (perhaps link to the stubs for now?)

File: _docs-sources/guides/welcome.md (hunk 1 above)
Comment (eak12913) [reply _docs-sources/guides/welcome.md:11]: We captured this when discussing it live

**Review by eak12913 — APPROVED** [review]
LGTM!

**Pull request merged**